Assets/AirConsole/examples/pong/ExamplePongLogic.cs

```diff
@@ -1,129 +1,240 @@
+    ///     For more information read: http://developers.airconsole.com/#/guides/device_ids_and_states
+    /// </summary>
+    /// <param name="device_id">The device_id that connected</param>
+    private void OnConnect(int device_id)
+    {
+        CheckTwoPlayers();
+    }
+
+    /// <summary>
+    ///     If the game is running and one of the active players leaves, we reset the game.
+    /// </summary>
+    /// <param name="device_id">The device_id that has left.</param>
+    private void OnDisconnect(int device_id)
+    {
+        var player = AirConsole.instance.ConvertDeviceIdToPlayerNumber(device_id);
+        if (player >= 0)
+            // Player that was in the game left the game.
+            // Setting active players to length 0
+            AirConsole.instance.SetActivePlayers(0);
+
+        CheckTwoPlayers();
+    }
+
+    /// <summary>
+    ///     We check for the start game command and otherwise which one of the active players has moved the paddle.
+    /// </summary>
+    /// <param name="from">From.</param>
+    /// <param name="data">Data.</param>
+    private void OnMessage(int device_id, JToken data)
+    {
+        if (data["start_game"] != null)
+        {
+            StartGame();
+            return;
+        }
+
+        var active_player = AirConsole.instance.ConvertDeviceIdToPlayerNumber(device_id);
+        if (active_player != -1)
+        {
+            if (active_player == 0) racketLeft.velocity = Vector3.up * (float)data["move"];
+
+            if (active_player == 1) racketRight.velocity = Vector3.up * (float)data["move"];
+        }
+    }
+    
+    private void OnPause()
+    {
+        Time.timeScale = 0;
+        // If we were playing any sounds we must mute them when the game gets paused
+        AudioListener.pause = true;
+    }
+
+    private void OnResume()
+    {
+        Time.timeScale = 1;
+        AudioListener.pause = false;
+    }
+
+    /// <summary>
+    /// Pause the game when Ad is shown to the player
+    /// </summary>
+    private void OnAdShow()
+    {
+        Time.timeScale = 0;
+        // If we were playing any sounds we must mute them when an ad is shown
+        AudioListener.pause = true;
+    }
+
+    /// <summary>
+    /// Resume the game after the Ad completed
+    /// </summary>
+    /// <param name="adShown"></param>
+    private void OnAdComplete(bool adShown)
+    {
+        Time.timeScale = 1;
+        AudioListener.pause = false;
+    }
+
+    /// <summary>
+    ///     Updates the displayed text on the screen and sets the device state property "enough_players"
+    ///     which allows the controller to react to the new device state
+    /// </summary>
+    private void CheckTwoPlayers()
+    {
+        var connectedControllers = AirConsole.instance.GetControllerDeviceIds();
+
+        // Only update if the game didn't have active players.
+        if (AirConsole.instance.GetActivePlayerDeviceIds.Count == 0)
```

Comment:
Shouldn't this be 

```suggestion
        if (AirConsole.instance.GetActivePlayerDeviceIds().Count == 0)
```

given `GetActivePlayerDeviceIds` is a method?

Reply:
No, not with the current implementation of AirConsole.cs

Reply:
Did honestly not realize this was still a property - I thought I had caught all properties before to either name them accordingly or make them functions ...
thanks for pointing that out.